pkg/cmd/roachtest/github.go

```diff
@@ -46,6 +47,26 @@ func roachtestPrefix(p string) string {
 	return "ROACHTEST_" + p
 }
 
+// generateHelpCommand creates a HelpCommand for createPostRequest
+func generateHelpCommand(
+	clusterName string, start time.Time, end time.Time,
+) func(renderer *issues.Renderer) {
+	return func(renderer *issues.Renderer) {
+		issues.HelpCommandAsLink(
+			"roachtest README",
+			"https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md",
+		)(renderer)
+		issues.HelpCommandAsLink(
+			"How To Investigate (internal)",
+			"https://cockroachlabs.atlassian.net/l/c/SSSBr8c7",
+		)(renderer)
+		issues.HelpCommandAsLink(
+			"Grafana",
+			fmt.Sprintf("https://go.crdb.dev/p/roachfana/%s/%d/%d", clusterName, start.Unix()*1000, end.Unix()*1000),
```

Comment:
for the whole Unix()*1000 code: it looks like the grafana needs nsec to be included in the link? 

my reasoning for doing this was:
- there is "no data" whenever i remove nsec (here is a link where the start and end times are just unix epoch seconds: https://go.crdb.dev/p/roachfana/silvano-cluster/1689957243/1689957733)
- i wasn't sure if doing something like:
```
startSec := start.Unix()*1000 + start.UnixNano()%1e+9
```

would have been overkill (feel free to let me know!)

Reply:
`start.UnixMilli()`? 

`Unix()` provides seconds

Reply:
is there a facepalm emoji? no way that i forgot milliseconds existed... thank you